.github/workflows/docker.yml

```diff
@@ -2,6 +2,8 @@ name: Publish Docker image and deploy
 on:
   release:
     types: [published]
+  push:
+    branches: [issue-50]
```

Comment:
need to remove the trigger before merging

Reply:
Did this run, because it is a in a fork I think it will not run. Should I merge it to a branch to test it properly?

Reply:
I think that would be the best idea 👍